test/index.js

```diff
@@ -30,4 +34,11 @@ function test() {
   });
   eventTarget.addEventListener('foo', () => console.log(2));
   eventTarget.dispatchEvent(new Event('foo'));
+  eventTarget.addEventListener('foo', added, {once: true});
+  eventTarget.addEventListener('foo', (event) => {
+    console.log(2);
+    event.stopImmediatePropagation();
+  });
+  eventTarget.addEventListener('foo', () => console.log(3), {once: true});
+  eventTarget.dispatchEvent(new Event('foo'));
```

Comment:
I did it that way for the PR to match current code, but for my local tests I copy/pasted linkedom `assert.js` file (+ some changes to make debugging easier since there is only EventTarget to test).

Reply:
agreed, tests were a bit nave in here ... as long as expectations are clear, and test small enough, I think this is fine.